docs/UserGuide.md

```diff
@@ -181,10 +195,19 @@ Format: `mark ANIMALINDEX TASKINDEX [TASKINDEX]...`
 * The `TASKINDEX` refers to the index of the task on the task list of the animal.
 * If multiple `TASKINDEX` are specified, all the tasks at the specified `TASKINDEX` will be marked as done.
 * `TASKINDEX` must be separated by a space.
+* If one of the `TASKINDEX` provided is invalid, the rest of the `TASKINDEX` will not be marked as done.
+* An error would not be thrown if the task(s) was previously marked as done, it would remain as marked.
```

Comment:
I think a better way to phrase it would be "Executing the `mark` command on task(s) previously marked as done will not cause an error to be thrown, and the task(s) would remain as marked."

Reply:
It does sound better! Ill change it